_posts/2020-05-27-ols1-wrapup.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: End of OLS-1 and OLS-2 Announcement post
+authors: 
+- yochannah
+- bebatut
+- malvikasharan
+image: https://images.unsplash.com/photo-1577036421869-7c8d388d2123?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80
+---
+
+*We are extremely delighted to announce that leads to 18 projects from our first cohort have graduated from the Open Life Science program.
```

Comment:
```suggestion
*We are extremely delighted to announce that leads to {{ grad-projects | size }} projects from our first cohort have graduated from the Open Life Science program.
```

Reply:
I think this will give 20 projects but in the end 2 people never finished. We might need to add a rule there?

Reply:
I see that you have added grad* tags <3